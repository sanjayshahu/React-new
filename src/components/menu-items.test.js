import { shallow,mount,render } from "enzyme";
import { idText } from "typescript";
import {React}from "react";
import MenuItems from "./menu-items";
import toJson from 'enzyme-to-json';
it('take snapshot',()=>{
    expect( toJson(shallow(<MenuItems/>))).toMatchSnapshot();//use this command for taking snapshot
}) 
