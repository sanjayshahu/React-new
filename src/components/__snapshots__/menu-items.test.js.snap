// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`take snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
